external-docs/docs/guides/command-line/inspect-command.md

```diff
@@ -99,7 +99,7 @@ In the following, the format of the outputs generated by the inspect command is
   | ------------------------------------- | ------------------------------------------------------------------------------------------------------------- |
   | Dataset Label                         | Label of the dataset.                                                                                         |
   | Number of Components                  | Number of components in the data structure definition.                                                        |
-  | Components                            | List of components with component property, property label, property type, column title and required columns. |
+  | Components                            | List of components with component property, property label, property type, column title, observation value column titles and required columns. |
```

Comment:
This is where running the [prettify markdown tables](https://marketplace.visualstudio.com/items?itemName=darkriszty.markdown-table-prettify) extension can be helpful.

Reply:
Done